packages/medusa/src/services/return-reason.js

```diff
@@ -92,6 +109,40 @@ class ReturnReasonService extends BaseService {
 
     return item
   }
+
+  async delete(returnReasonId) {
+    return this.atomicPhase_(async manager => {
+      const rrRepo = manager.getCustomRepository(this.retReasonRepo_)
+
+      // We include the relation 'return_reason_children' to enable cascading deletes of return reasons if a parent is removed
+      const reason = await this.retrieve(returnReasonId, {relations: ["return_reason_children"]})
```

Comment:
Please check that your linter/prettier is correctly configured. This line is more than 80 cols wide so it should be split into multiple lines